app/tests/spec/lib/metrics.js

```diff
@@ -74,11 +80,18 @@ function (chai, $, p, Metrics, AuthErrors, WindowMock, TestHelpers) {
         assert.equal(filteredData.migration, 'sync1.5');
         assert.equal(filteredData.campaign, 'fennec');
 
+        assert.equal(filteredData.referrer, 'https://marketplace.firefox.com');
```

Comment:
A bit of a tangential addition. `referrer` was on the list of allowed fields, but did not have a test.